clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemUtils.java

```diff
@@ -134,4 +142,42 @@ public static boolean waitCompleted(final TachyonFileSystemCore tfs, final Tachy
 
     return completed;
   }
+
+  /**
+   * Persist the given file to the under file system.
+   *
+   * @param tfs {@link TachyonFileSystem} to carry out tachyon operations
+   * @param file the file to persist
+   * @param fileInfo the file info of the file
+   * @param tachyonConf TachyonConf object
+   * @return the size of the file persisted
+   * @throws IOException if an I/O error occurs
+   * @throws FileDoesNotExistException if the given file does not exist
+   * @throws TachyonException if an unexpected Tachyon error occurs
+   */
+  public static long persistFile(TachyonFileSystem tfs, TachyonFile file, FileInfo fileInfo,
```

Comment:
I had a question about this utility method. Non-shell clients can call this, right? If so, is it strange that you have to give it a File and a FileInfo for that file?

Also, would the TachyonConf always be the conf from ClientContext?